src/iop/gaussian_elimination.h

```diff
@@ -21,7 +21,7 @@
     equations by Gaussian elimination with partial pivoting.  The
     functions gauss_make_triangular and gauss_solve have been adopted
     from Fortran routines as presented in the book "Numerik" by
-    Helmuth Späth, Vieweg Verlag, 1994, see also
+    Helmuth Spaeth, Vieweg Verlag, 1994, see also
```

Comment:
This was to avoid UTF-8 encoding of the file; I can revert it for the comment if you wish, but other changes removing UTF-8 characters in the error printf below should stay I think.

Reply:
Well all our files should be UTF-8 anyway. So yes I would revert this. TIA.

Reply:
Was reverted already.